pkg/render/render.go

```diff
@@ -151,6 +152,21 @@ func selectedIf(v bool) htmltemplate.HTML {
 	return ""
 }
 
+// translate accepts a message printer (populated by middleware) and prints the
+// translated text for the given key. If the printer is nil, an error is
+// returned.
+func translate(l *gotext.Locale, key string, vars ...interface{}) (string, error) {
```

Comment:
This is important because the default behavior is to just return the empty string, which is a pretty difficult-to-debug failure. I'd like the template execution to fail on missing i18n keys.